assets/eip-5216/README.md

```diff
@@ -0,0 +1,51 @@
+<div align="center">
+
+# ERC1155 Approval By Amount Extension
+
+[![License: MIT](https://img.shields.io/badge/License-MIT-yellow.svg)](https://opensource.org/licenses/MIT)
```

Comment:
Everything in this repository must be CC0 licensed.

Reply:
I'll delete everything except a .sol with reference implementation.

Also I will delete test cases. I'll maintain a repository with a complete project with deps, a build system and test infrastructure in my personal account.

Reply:
Looks much better now, thanks!  Also Note: Test Cases and Reference Implementation are both optional for ERCs, so if in the future you are struggling to come up with a trimmed down set of test cases or a trimmed down reference implementation you *can* leave them out entirely.  Generally it is better to include them if at all possible.

Reply:
Hello @MicahZoltu, thank you for everything.

Of course I will do it. In the .sol I left the example token as a reference implementation, but for the test cases, I would need an example to look at to make my test cases according to the style of this magnificent repository.

So, do you think it can advance to a next step this eip?

Reply:
I'm not an ERC reviewer, so you'll have to wait for one of the people the bot mentioned to review this before it can get merged.  I just try to help out from time to time by providing feedback to authors so they don't get held up unnecessarily.

Reply:
Thank you very much then @MicahZoltu. Appreciated 😁